src/com/google/enterprise/adaptor/prebuilt/FilterMimetypes.java

```diff
@@ -131,14 +131,14 @@ public void transform(Metadata metadata, Map<String, String> params) {
     ct = ct.trim().toLowerCase();
     String decision = lookupDecision(ct);
     if (null != decision) {
-      params.put("Transmission-Decision", decision);
+      if (!decision.equals(TransmissionDecision.AS_IS.toString())) {
+        params.put("Transmission-Decision", decision);
```

Comment:
While we're reindenting this line, maybe use MetadataTransform.KEY_TRANSMISSION_DECISION as the deleted lines below did?